src/main/c/sslcontext.c

```diff
@@ -1340,16 +1340,6 @@ TCN_IMPLEMENT_CALL(void, SSLContext, setSessionTicketKeys)(TCN_STDARGS, jlong ct
     int i;
     int cnt;
 
-    if (((*e)->GetArrayLength(e, keys) % SSL_SESSION_TICKET_KEY_SIZE) != 0) {
```

Comment:
This method does a few direct memory operations (memcpy) based upon the assumption that this conditional holds. Should this method be responsible for validating this conditional instead of hoisting the responsibility on the caller? Seems like unexpected results may happen (that may be tough to track down) if the caller forgets to validate this.